.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         toolchain:
           - stable
             # msrv
-          - 1.56.1
```

Comment:
`cargo msrv` is reporting `1.66.1` as the msrv. I'd move this number to that one.